aten/src/ATen/core/DeprecatedTypeProperties.h

```diff
@@ -41,8 +50,8 @@ class DeprecatedTypeProperties {
     return scalarTypeToTypeMeta(scalar_type_);
   }
 
-  bool is_defined() const {
-    return backend_ != Backend::Undefined && scalar_type_ != ScalarType::Undefined;
+  bool is_variable() const {
+    return is_variable_;
```

Comment:
What happened here?

Reply:
is_defined() wasn't something on Type before, I was using it as a helper but looks like I actually don't need it.

is_variable() is something I want to move over.